docs/adr/adr-0018-how-to-give-feedback-after-a-user-action.md

```diff
@@ -0,0 +1,414 @@
+| NVDA + Chrome     | ✅ Fully supported        | ✅ Fully supported          | ❌ Not supported           | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| JAWS + Chrome     | ✅ Fully supported        | ✅ Fully supported          | ❌ Not supported           | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| JAWS + EDGE       | ✅ Fully supported        | ✅ Fully supported          | ❌ Not supported           | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| JAWS + Firefox    | ❌ Not supported          | ✅ Fully supported          | ⚠ Partially supported (1) | ✅ Fully supported         | ✅ Fully supported                        | ❌ Not supported                  | ✅ Fully supported                       |
+| VO + Safari (Mac) | ✅ Fully supported        | ✅ Fully supported          | ✅ Fully supported         | ⚠ Partially supported (2) | ⚠ Partially supported (2)                | ✅ Fully supported                | ✅ Fully supported                       |
+| VO + Safari (iOS) | ✅ Fully supported        | ✅ Fully supported          | ✅ Fully supported         | ✅ Fully supported         | ✅ Fully supported                        | ✅ Fully supported                | ✅ Fully supported                       |
+| Talkback + Chrome | ✅ Fully supported        | ✅ Fully supported          | ✅ Fully supported         | ✅ Fully supported         | ✅ Fully supported                        | ⚠ Partially supported (3)        | ✅ Fully supported                       |
+
+**Note** :
+* ✅ Fully supported means the aria-live and aria-atomic settings are respected. However, most of the time, **role semantics are not announced**. We consider this a very minor issue in our case because we already convey this semantic through an icon with an alternative text.
+* ⚠ Partially supported means that we found issues that makes this solution not robust enough for our use case. Usually, something is announced, but it does not respect the aria-atomic setting.
+* ❌ Not supported means nothing is announced.
+
+(1) sometimes not announced at all
+
+(2) wrong announcement
+
+(3) atomicity is not applied
+
+### Comparing our implementation with other libraries
+
+It's always interesting to check differences with other libs.
+We took two different toasters:
+
+* https://fkhadra.github.io/react-toastify/
+  * this one works only with React
+  * There are a lot of features that we don't need
+  * It is 6.5 kb code (https://bundlephobia.com/package/react-toastify@9.0.6)
+* https://github.com/Maronato/vue-toastification
+  * This one works only with Vue.js
+  * It is 6,1 kB of code https://bundlephobia.com/package/vue-toastification@1.7.14
```

Comment:
I'm wondering if size info works because readers don't know the size of our toast so they can't really compare

Reply:
We can add some details about the size of our own system?

Reply:
6,1 kB => 6.1 kB

Reply:
I just checked from the CDN: our implementation is around 5 kB for both components but it brings Lit + the i18n system of course. So maybe we should not mention it, I don't know.

Reply:
The react version also needs a 3 kB CSS file. Same for the Vue version. Whereas our 5 kB contain almost eveything (w eneed to add the theme).

Reply:
Not easy to compare, so I'll remove the sizes. And it is not really our point here.